geonetwork-api/src/test/java/au/org/aodn/geonetwork_api/openapi/api/helper/SiteHelperTest.java

```diff
@@ -0,0 +1,52 @@
+package au.org.aodn.geonetwork_api.openapi.api.helper;
+
+import au.org.aodn.geonetwork_api.openapi.api.SiteApi;
+import au.org.aodn.geonetwork_api.openapi.model.Setting;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.springframework.http.ResponseEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+public class SiteHelperTest {
+
+    /**
+     * Function test only, we do not have docker image to run for this test, the
+     * integration test in geonetwork should test it more.
+     */
+    @Test
+    public void verifyGetAllSettingsDetails() {
+        Setting s1 = new Setting();
+        s1.setName(SiteHelper.HOST);
+        s1.setValue("geonetwork-edge.edge.aodn.org.au");
```

Comment:
can the value be managed in the test's properties yaml?

and you can create a variable for being reused at line 49.

Reply:
Introduce constant